### DISCUSSION TITLE
Support HTTPS for importers

### DISCUSSION THREAD

**Discussion by zifeo** [firefly-iii/firefly-iii#4389] [Firefly III] [answered]
**Description**

Currently, the CSV importer cannot be used with HTTPS as static files (JS/CSS) are served through HTTP (mixed content). Some redirections (oath workflow?) might also be affected.

**Solution**

Change `href="http://` by `href="//` and similar tricks.

**What are alternatives?**

None secured (stick with HTTP).

**Comment by JC5** [comment]
The CSV importer supports https. You have to configure your reverse proxy correctly.

**Reply by JC5** [reply]
https://docs.firefly-iii.org/csv/errors/freq_errors/#i-get-page-load-errors-because-the-protocols-dont-match

**Comment by zifeo** [comment, marked as answer]
@JC5 Thanks for the help, was already managed. In fact the TRUSTED_PROXIES set to "**" was missing. Curious to see how Laravel resolves those headers instead of letting the browser do it.
